data/json/monsters/zed_misc.json

```diff
@@ -276,6 +278,8 @@
     "//": "the headless horror is technically blind, but its feelers allow it to sense your presence if you're close to it",
     "vision_day": 2,
     "vision_night": 2,
+    "weakpoint_sets": [ "wps_humanoid_body" ],
```

Comment:
Needs a new weakpoint set that doesn't include a head critical.

Reply:
Done